custom/ewsghana/tasks.py

```diff
@@ -189,6 +190,13 @@ def migrate_email_settings(domain):
     endpoint = GhanaEndpoint.from_config(config)
     migrate_email = EmailSettingsSync(domain)
 
+    for user in iter_docs(WebUser.get_db(), WebUser.ids_by_domain(domain)):
+        for notification in ReportNotification.by_domain_and_owner(domain, user['_id'], stale=False):
+            notification.delete()
```

Comment:
Actually, is it possible to not delete these scheduled reports but just stop them from being sent?